service/src/main.rs

```diff
@@ -334,6 +336,42 @@ fn start_worker<E, T, D>(
 	let mut xthex = hex::encode(uxt);
 	xthex.insert_str(0, "0x");
 
+	//Account funds
+	if dev {
+		//dev mode, the faucet will ensure that the enclave has enough fund
+		if let Err(x) = ensure_account_has_funds(&mut node_api, &tee_accountid) {
+			error!("Starting worker failed: {:?}", x);
+			return
+		}
+	} else {
+		//production mode, there is no faucet.
+		let registration_fees = match enclave_registration_fees(&mut node_api, &xthex.clone()) {
+			Err(x) => {
+				error!("Starting worker failed: {:?}", x);
+				return
+			},
+			Ok(f) => f,
+		};
+		info!("Registration fees = {:?}", registration_fees);
+		let free = node_api.get_free_balance(&tee_accountid).unwrap();
+		info!("TEE's free balance = {:?}", free);
+
+		let missing_funds = registration_fees.saturating_mul(MIN_FUND_FACTOR).saturating_sub(free);
+		info!("Missing funds to continue ... : {:?}", missing_funds);
+
+		if missing_funds > 0 {
+			//if not enough funds,then the user can send the missing TEER to the enclave adress and start again
```

Comment:
In general: Comments should be correct sentences with punctuation and a space between the `//` and the comment.
```suggestion
			// If there are not enough funds, then the user can send the missing TEER to the enclave adress and start again.
```